cmd/commands/dev.go

```diff
@@ -13,16 +13,14 @@ import (
 func NewCmdDev() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "dev",
-		Short:   "Run the inngest dev server",
-		Example: "inngest dev -p 1234",
```

Comment:
This shows the recommended default that we want people to use now

Reply:
Yeah great idea.